[2021-02-17 03:23:41,512] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: tutorial_etl_dag.transform 2021-02-17T03:23:09.230964+00:00 [queued]>
[2021-02-17 03:23:41,584] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: tutorial_etl_dag.transform 2021-02-17T03:23:09.230964+00:00 [queued]>
[2021-02-17 03:23:41,585] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-02-17 03:23:41,586] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-02-17 03:23:41,586] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-02-17 03:23:41,650] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): transform> on 2021-02-17T03:23:09.230964+00:00
[2021-02-17 03:23:41,664] {standard_task_runner.py:52} INFO - Started process 208 to run task
[2021-02-17 03:23:41,670] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'tutorial_etl_dag', 'transform', '2021-02-17T03:23:09.230964+00:00', '--job-id', '70', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/tutorial_etl_dag.py', '--cfg-path', '/tmp/tmpge9ikco7', '--error-file', '/tmp/tmp6f_rjxgv']
[2021-02-17 03:23:41,675] {standard_task_runner.py:77} INFO - Job 70: Subtask transform
[2021-02-17 03:23:41,786] {logging_mixin.py:104} INFO - Running <TaskInstance: tutorial_etl_dag.transform 2021-02-17T03:23:09.230964+00:00 [running]> on host 63b50af6ef7d
[2021-02-17 03:23:41,888] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=tutorial_etl_dag
AIRFLOW_CTX_TASK_ID=transform
AIRFLOW_CTX_EXECUTION_DATE=2021-02-17T03:23:09.230964+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-02-17T03:23:09.230964+00:00
[2021-02-17 03:23:41,938] {python.py:118} INFO - Done. Returned value was: None
[2021-02-17 03:23:41,971] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=tutorial_etl_dag, task_id=transform, execution_date=20210217T032309, start_date=20210217T032341, end_date=20210217T032341
[2021-02-17 03:23:42,038] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-02-17 03:23:42,089] {local_task_job.py:146} INFO - Task exited with return code 0
